src/theory/strings/array_solver.h

```diff
@@ -54,6 +55,33 @@ class ArraySolver : protected EnvObj
    * their application to concatenation terms.
    */
   void checkArrayConcat();
+  /**
+   * Perform reasoning about seq.nth and seq.update operations (lazily), which
+   * calls the core array solver for the set of nth/update terms over atomic
```

Comment:
```suggestion
   * calls the core sequences-array solver for the set of nth/update terms over atomic
```
Just so it doesn’t sound like we are calling the “real” AF_AX solver.